.github/workflows/ci-artifacts.yml

```diff
@@ -164,3 +176,91 @@ jobs:
           cat unzip-test.out &&
           test "grep is /usr/bin/grep" = "$(type grep)" || exit 1
           grep README unzip-test.out
+  publish-release-assets:
+    if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+    runs-on: ubuntu-latest
+    needs: [test-minimal-sdk, assorted-validations]
+    steps:
+      - name: download minimal-sdk artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: minimal-sdk
+          path: ${{github.workspace}}
+      - name: download minimal-sdk artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: minimal-sdk-extra
+          path: ${{github.workspace}}
+      - name: publish release asset
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const req = { owner: context.repo.owner, repo: context.repo.repo }
+            // find or create the GitHub release named `ci-artifacts`
+            const release = await (async () => {
+              try {
+                return await github.rest.repos.getReleaseByTag({ ...req, tag: 'ci-artifacts' });
+              } catch (e) {
+                if (e.status === 404) {
+                  // create the `ci-artifacts` GitHub release based on the current revision
+                  const workflowRunsURL = `${context.serverUrl}/${
+                    process.env.GITHUB_WORKFLOW_REF.replace(/\/\.github\/workflows\/([^@]+).*/, '/actions/workflows/$1')
+                  }`
+                  return await github.rest.repos.createRelease({
+                    ... req,
+                    tag_name: 'ci-artifacts',
+                    body: `Continuous release of \`ci-artifacts\`
+
+            This release is automatically updated by the [ci-artifacts](${workflowRunsURL}) workflow.
+
+            For technical reasons, allow up to a minute for release assets to be missing while they are updated.`,
+                  });
+                }
+                throw e;
+              }
+            })()
+
+            const fs = require('fs')
+            for (const fileName of [
+              'git-sdk-x86_64-minimal.tar.gz',
+              'git-sdk-x86_64-minimal.zip',
+              'git-sdk-x86_64-minimal.7z.exe',
+            ]) {
```

Comment:
[6 seconds](https://github.com/dscho/git-sdk-64/actions/runs/10851317340/job/30115544965#step:4:81) is already really fast, but I wonder if we want to handle these uploads in parallel to potentially speed things up further. But it's a really minor nit, so please feel free to ignore!

Reply:
This is a once-a-day job, so I'm fine with it running a little slower than absolutely necessary ;-)